src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent.mdx

```diff
@@ -71,6 +71,13 @@ To install the Node.js agent:
   ```
   node -r newrelic ./dist/server.js
   ```
+  
```

Comment:
it feels like this should be its own page.  At the very least further down with a heading in here.  This is breaking up the -r vs require inline copy

Reply:
I can create a new page for it, with content that's more like the more-complete example README.